iap_alternative_provider/views/iap_account_view.xml

```diff
@@ -7,14 +7,17 @@
             <xpath expr="//group[@name='account']" position="before">
                 <group string="Info" name="info">
                     <field name="provider" />
-                    <field name="name" />
-                    <field name="company_ids" position="move" />
+                    <field name="name" position="move" />
```

Comment:
the move has no sense IMO...

Reply:
agreed, if `name`  is removed on the model side, it can be removed too on the xml side.

Reply:
If you believe name should be removed from form view I'm happy to do it, just clarify please (request change) so we can move on with the merge.

Reply:
The move was removing the field from the view. Just remove the line.